drivers/gpio/gpio_rv32m1.c

```diff
@@ -34,25 +34,72 @@ struct gpio_rv32m1_data {
 	u32_t pin_callback_enables;
 };
 
+static u32_t get_port_pcr_irqc_value_from_flags(struct device *dev,
```

Comment:
This function is not used anymore by `gpio_rv32m1_configure`. We could move it directly above `gpio_rv32m1_pin_interrupt_configure`. It would make the code easier to read.